### PR TITLE
feat(ui): A help message for managing multiple ships in the outfitter

### PR DESCRIPTION
**Pull request by TomGoodIdea**
## Summary
Added a help message that appears when you open the outfitter with more than one ship. It explains that you can change outfits of multiple ships at once and also tells you about the "has this outfit installed" sidebar indicators.

## Testing Done
Works both automatically and manually.

## Performance Impact
N/A

### CODE REVIEW

**Review by quyykk — APPROVED** [review]

File: data/_ui/help.txt [hunk 1]
```diff
@@ -161,6 +161,10 @@ help "uninstalling and storage"
...
 	`If you sell an outfit at an outfitter, then you can immediately buy it back for the same price that you sold it, so long as you do not leave the planet. This means that you do not necessarily need to worry about uninstalling outfits into planetary storage when you are changing outfits around.`
 
+help "outfitter with multiple ships"
+	`In the outfitter you can manage multiple ships at the same time. Ctrl+click a ship icon to add it to your selection. You can also use Shift+click to select a range of ships.`
+	`When you have more than one ship selected, you can install or remove outfits from all of those ships at once. Additionally, all selected ships that have the currently selected outfit installed are marked with a pointer next to their sidebar icon.`
```
Comment (mOctave) [inline data/_ui/help.txt:166]: Okay as is, but a bit wordy. Could probably be pruned even more than this, honestly.

```suggestion
	`When you have more than one ship selected, you can install or remove outfits from all of those ships at once. Additionally, all selected ships with the currently selected outfit installed are marked with a pointer next to their sidebar icon.`
```

Also, is there a better name for this than "sidebar icon"?

**Review by OcelotWalrus — APPROVED** [review]

**Review by mOctave — APPROVED** [review]

File: source/OutfitterPanel.cpp [hunk 2]
```diff
@@ -101,7 +101,8 @@ void OutfitterPanel::Step()
 	if(GetUI()->IsTop(this) && !checkedHelp)
 		// Use short-circuiting to only display one of them at a time.
 		// (The first valid condition encountered will make us skip the others.)
-		if(DoHelp("outfitter") || DoHelp("cargo management") || DoHelp("uninstalling and storage") || true)
+		if(DoHelp("outfitter") || DoHelp("cargo management") || DoHelp("uninstalling and storage")
+				|| (player.Ships().size() > 1 && DoHelp("outfitter with multiple ships")) || true)
```
Comment (warp-core) [inline source/OutfitterPanel.cpp:105]: Would it not make more sense to only show this message if the player has more than one ship *here*?

File: data/_ui/help.txt [hunk 3]
```diff
@@ -161,6 +161,10 @@ help "uninstalling and storage"
 	`You can also uninstall outfits directly into planetary storage by selecting the outfit and pressing U. This can be useful for if you want to shuffle outfits between ships, store outfits for future use, or simply try on different outfits. Outfits in storage can also be transferred to cargo when you go to 'purchase to cargo' mode. Outfits that are in planetary storage or cargo can be (re)installed at any outfitter without cost.`
 	`If you sell an outfit at an outfitter, then you can immediately buy it back for the same price that you sold it, so long as you do not leave the planet. This means that you do not necessarily need to worry about uninstalling outfits into planetary storage when you are changing outfits around.`
 
+help "outfitter with multiple ships"
+	`In the outfitter you can manage multiple ships at the same time. Ctrl+click a ship icon to add it to your selection. You can also use Shift+click to select a range of ships.`
```
Comment (warp-core) [inline data/_ui/help.txt:165]:
```suggestion
	`You can manage outfits on multiple ships at the same time. Ctrl+click a ship icon to add it to your selection. You can also use Shift+click to select a range of ships.`
```
It should be fairly clear by now that this is the outfitter and "manage" doesn't alone necessarily imply "outfits" in my opinion. When I think of "managing" ships, I think of the player info panel and adjusting the order or parking status of ships.
